mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/support/DefaultProtocolMethodProcessorImpl.java

```diff
@@ -116,9 +120,10 @@ public void processConnect(Channel channel, MqttConnectMessage msg) {
 
         // Check MQTT protocol version.
         if (msg.variableHeader().version() != MqttVersion.MQTT_3_1.protocolLevel()
-                && msg.variableHeader().version() != MqttVersion.MQTT_3_1_1.protocolLevel()) {
+                && msg.variableHeader().version() != MqttVersion.MQTT_3_1_1.protocolLevel()
+                && msg.variableHeader().version() != MqttVersion.MQTT_5.protocolLevel()) {
```

Comment:
Would we add a new method : isSupportedVersion() in MqttUtils?

Reply:
Ok, i will fix it ASAP